app/scripts/metamask-controller.js

```diff
@@ -1485,7 +1485,12 @@ export default class MetamaskController extends EventEmitter {
           'NotificationServicesController:disableNotificationServices',
           'NotificationServicesController:selectIsNotificationServicesEnabled',
         ],
-        allowedEvents: ['KeyringController:lock', 'KeyringController:unlock'],
+        allowedEvents: [
+          'KeyringController:lock',
+          'KeyringController:unlock',
+          'AccountsController:accountAdded',
```

Comment:
Oh you had to add those! Wasn't sure you would need to before I activated account syncing. Thanks!

Reply:
Yep test oddly expect the correct allowed events need to be provided.
All good. I am concerned that this controller will contain many events in the future - we can worry/split this out later though.